src/components/merchantList/merchantList.css

```diff
@@ -0,0 +1,61 @@
+.merchant-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(360px, 1fr));
+}
+.merchant-title {
+  color: #3f3e3e;
+  font-size: 16px;
+  padding: 10px 0;
+}
+.merchant-list__content {
+  display: flex;
+  gap: 14px;
+  align-items: center;
+  padding: 20px 0;
+}
+.merchant-list__content-image-item {
+  width: 85px;
+  height: 85px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+.merchant-list__content-info-name {
+  color: #3e3e3e;
+  font-size: 14px;
+  font-weight: 500;
+  text-overflow: ellipsis;
+}
+.merchant-list__content-info-first-yellow {
+  color: #fab125;
+  font-size: 12px;
+  font-weight: 700;
+}
+.merchant-list__content-info-first-dark {
+  font-weight: 300;
+  color: #717171;
+  font-size: 0.75rem;
+  line-height: 0.9375rem;
```

Comment:
just a doubt, why do you are using this value `0.9375` how this calculation was done to use this value? 🤔